### DISCUSSION TITLE
Custom installation directory?

### DISCUSSION THREAD

**Discussion by amezhong1** [sebdelsol/sfvip-all#19] [Ideas]
Can we have the option to customize the directory which sfvip-all is installed to? I like to keep my software in a specific folder, and every time sfvip-all updates it installs to the default directory. Thank you!

**Comment by sebdelsol** [comment]
If I remember correctly there's a way to do that with NSIS. I'll have to check.

**Comment by sebdelsol** [comment]
[_Sfvip All **1.4.12.43** installer_](https://github.com/sebdelsol/sfvip-all#downloads) has an option to modify the install directory.

![Capture d’écran 2024-03-25 224211](https://github.com/sebdelsol/sfvip-all/assets/6039281/a2a17ea3-4b85-415f-80de-f5c9ed7f4654)

**N.B.**:
* It suggests the last installed version directory as default.
* It silently uninstalls the currently installed version if it's in a different directory[^1] .
* When updating from _Sfvip All_ it updates & runs in its current installed directory[^2].

[^1]:So you won't end up with different _Sfvip All_ installs on your system.
[^2]:The only way to change the install directory is to run the installer.

**Comment by amezhong1** [comment]
Awesome, thank you so much!

**Comment by sebdelsol** [comment]
I fixed a minor bug in **v1.4.12.44** : 
✅ old installs are now correctly uninstalled even when it's running.
You should see the update soon when you open the [settings](https://github.com/sebdelsol/sfvip-all#settings).
